mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -218,23 +232,30 @@ def remove_repeated_time_index(ds): #{{{
     # remove repeated indices
     ds = ds.isel(Time=index)
 
-    return ds #}}}
+    return ds  # }}}
 
-def test_load_mpas_xarray_datasets(path): #{{{
-    ds = xr.open_mfdataset(path, preprocess=lambda x: preprocess_mpas(x, yearoffset=1850))
+
+def test_load_mpas_xarray_datasets(path):  # {{{
+    ds = xr.open_mfdataset(path, preprocess=lambda x:
+                           preprocess_mpas(x, yearoffset=1850))
     ds = remove_repeated_time_index(ds)
 
     # make a simple plot from the data
     ds.Time.plot()
     plt.show()
 
-    return #}}}
+    return  # }}}
+
 
-def test_load_mpas_xarray_timeSeriesStats_datasets(path): #{{{
-    ds = xr.open_mfdataset(path, preprocess=lambda x: preprocess_mpas(x,
-        timeSeriesStats=True, timestr='timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1'))
+def test_load_mpas_xarray_timeSeriesStats_datasets(path):  # {{{
+    timestr = 'timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1'
```

Comment:
I'm not sure that change fit in the scope of this PR.  I'm just trying to make the line PEP8 compliant, not add any new functionality.  I pulled out timestr simply because the variable name is so long that there's no feasible way of keeping the line under 80 characters without assigning it to a variable.  

If you want to upgrade the testing of mpas_xarray, that would be a fine thing to do in another PR.